app/models/concerns/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] From the previous reviewer's screenshots, this file should be located in the `app/models` directory, not `app/models/concerns`.